ihatemoney/forms.py

```diff
@@ -319,33 +321,31 @@ class BillForm(FlaskForm):
     submit = SubmitField(_("Submit"))
     submit2 = SubmitField(_("Submit and add a new one"))
 
+    def export(self, project):
+        return Bill(
+            float(self.amount.data),
+            self.date.data,
+            self.external_link.data,
+            str(self.original_currency.data),
+            Person.query.get_by_ids(self.payed_for.data, project),
+            self.payer.data,
+            project.default_currency,
+            self.what.data,
+        )
```

Comment:
It could be good to also use named parameters here for consistency

Reply:
Oops, forgot to include them here, fixed !